index.html

```diff
@@ -4,7 +4,8 @@
 
 <head>
     <meta charset="utf-8">
-    <title>FOSSASIA Summit 2019 | Asia's Open Technology Event about AI, Personal Assistants, Blockchain ,Machine Learning, Open Hardware, Open Source Software, Open Design, Open Data in Singapore</title>
+    <title>FOSSASIA SUMMIT 2019</title>
+    <meta name="description" content="Asia's Open Technology Event about AI, Personal Assistants, Blockchain ,Machine Learning, Open Hardware, Open Source Software, Open Design, Open Data in Singapore">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="FOSSASIA Summit 2019 Singapore - Open Source, Big Data, Design Thinking and Free Knowledge Tools at Asia's Premier Open Technology Event">
     <meta name="keywords" content="FOSSASIA Summit,FOSS, Free and Open Source Software,Singapore, Asia">
```

Comment:
Looks like we have two meta tags

Reply:
Will I have to do in one description meta tag ?

Reply:
If you can merge both the tags it would be great.

Reply:
Ya I did, merge both the tags.... Please have a look.